app/tests/unit/routes/v1/merge.spec.js

```diff
@@ -72,6 +63,7 @@ describe(`POST ${basePath}`, () => {
   });
 
   it('should push a message and yield an Ethereal url', async () => {
+    ChesService.prototype.sendEmailMerge.mockResolvedValue('https://example.com');
```

Comment:
Same as the email comment, the mock service call needs to handle ethereal true/false, so we can actually test the route is doing the work figuring out if we are in devMode.  Have one with devMode=true and another with devMode spelled incorrectly and it should return a "normal" response.

Reply:
We can actually satisfy checking whether the sendEmailMerge function was invoked to run with ethereal as true or false by employing a jest spy. Assigning the spy to the mock, we can check what arguments were passed into the mocked function.
expect(spy).toHaveBeenCalledWith(undefined, body, true);